pkgs/by-name/fr/fresh-editor/package.nix

```diff
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  fetchurl,
+  pkg-config,
+  openssl,
+  gzip,
+  git,
+  makeBinaryWrapper,
+  writableTmpDirAsHomeHook,
+}:
+
+let
+  # Pre-fetch the V8 binary that rusty_v8 wants to download
+  rusty_v8_archive = fetchurl {
+    url = "https://github.com/denoland/rusty_v8/releases/download/v142.2.0/librusty_v8_release_x86_64-unknown-linux-gnu.a.gz";
+    hash = "sha256-xHmofo8wTNg88/TuC2pX2OHDRYtHncoSvSBnTV65o+0=";
+  };
```

Comment:
This will fail on aarch64-linux and on non-linux platforms. There are multiple packages on nixpkgs that also depend on `rusty_v8`, and `deno` actually exports the archive so it can be used in packages (although most packages seem to copy the fetcher expression). Search for `fetchLibrustyV8` for examples.